### PR TITLE
moduleForAcceptance codemod

### PR DESCRIPTION
**Pull request by snewcomer**
@rwjblue Added a simple codemod for `setupApplicationTest` as well as a note to the readme that users should also use `ember-test-helpers-codemod`.

### CODE REVIEW

**Comment by rwjblue** [comment]
restarted CI (failure seemed completely unrelated)

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "eslint": "^4.1.0",
     "eslint-config-prettier": "^2.6.0",
     "eslint-plugin-prettier": "^2.3.1",
-    "jest": "^21.0.0",
+    "jest": "^21.2.1",
```
Comment (rwjblue) [inline package.json]: Is 21.2.1 published?  CI keep choking on this...

File: yarn.lock [hunk 2]
```diff
@@ -2432,9 +2432,9 @@ jest-validate@^21.2.1:
     leven "^2.1.0"
     pretty-format "^21.2.1"
 
-jest@^21.0.0:
+jest@^21.2.1:
   version "21.2.1"
-  resolved "https://registry.yarnpkg.com/jest/-/jest-21.2.1.tgz#c964e0b47383768a1438e3ccf3c3d470327604e1"
+  resolved "https://artifacts.apple.com/api/npm/npm-amp/jest/-/jest-21.2.1.tgz#c964e0b47383768a1438e3ccf3c3d470327604e1"
```
Comment (rwjblue) [inline yarn.lock]: @snewcomer - This seems wrong, can you fix it plz?

Comment (rwjblue) [reply yarn.lock]: This is probably due to a global `.npmrc` or something...

Comment (snewcomer) [reply yarn.lock]: welp that is embarrassing.  yep npmrc change.

**Pull request merged**